susemanager-utils/testing/docker/master/Makefile

```diff
@@ -1,7 +1,7 @@
 VERSION = 4.2.0
 REGISTRY = registry.mgr.suse.de
 SHELL := /bin/bash
-BUILDOPTS=
+BUILDOPTS=-m 8g
```

Comment:
Why we need this limit?

Reply:
In the past I had the following phenomenom: When I build the container in a host with a lot of RAM (e.g. 32GB) and run it on a host with much less RAM (e.g. 8GB) the postgresql containers could not be started.
AFAIK the postgresql config values are alligned with the available memory values when the container is build. When we run it on a different host with less memory, it cannot reserve the requested memory and failed to start.